.github/workflows/on_push-release.yml

```diff
@@ -16,6 +16,9 @@ jobs:
     runs-on: macos-latest
     #    runs-on: ubuntu-20.04-16core # Larger github runner, with KVM acceleration
     permissions:
+      # Set permissions for ${{ secrets.GITHUB_TOKEN }}
+      # https://docs.github.com/en/actions/security-guides/automatic-token-authentication
+      packages: read
```

Comment:
```suggestion
      contents: read
      packages: read
```

Reply:
This has now been done; set to `contents: read`